apps/docs/pages/guides/storage/image-transformations.mdx

```diff
@@ -59,6 +59,32 @@ supabase.storage.from('bucket').download('image.jpg', {
 })
 ```
 
+## Automatic Image Optimisation Detection (WebP)
```

Comment:
Thoughts on just calling this "Automatic Image Optimization?" "Detection" seems a bit redundant.